doc/sphinx/addendum/universe-polymorphism.rst

```diff
@@ -529,8 +529,8 @@ sections, except in the following ways:
        Polymorphic Universe i.
        Fail Constraint i = i.
 
-  This includes constraints implictly declared by commands such as
-  :cmd:`Variable`, which may as a such need to be used with universe
+  This includes constraints implicitly declared by commands such as
+  :cmd:`Variable`, which may as such need to be used with universe
```

Comment:
Something had to be fixed, but not fully sure about the correct fix ("which may thus require to be used ..."?).

Reply:
Seems fine.  I think you could drop "as such".  I don't think it adds anything.

Reply:
I was also thinking at dropping "as such" but kept it in doubt.

I made the change, thanks.